manifests/config.pp

```diff
@@ -22,15 +21,72 @@
     mode    => '0600',
   }
 
-  file {'/etc/httpd/conf.d/pulp.conf':
```

Comment:
If you're moving to using ```apache::vhost``` (which I think is a good move), then you should remove the template (```templates/etc/httpd/conf.d/pulp.conf.erb```) as well.